examples/python_large_output/download_output_file.py

```diff
@@ -0,0 +1,93 @@
+# Copyright (C) 2022 - 2024 ANSYS, Inc. and/or its affiliates.
+# SPDX-License-Identifier: MIT
+#
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+"""
+Example to query resources from a project.
+
+- Query values from evaluated jobs, computing some simple statistics on parameter values.
+- Download files from the project
+
+"""
+import argparse
+import logging
+import os
+import time
+
+from ansys.hps.client import Client, HPSError
+from ansys.hps.client.jms import JmsApi, ProjectApi
+
+log = logging.getLogger(__name__)
+
+
+def download_files(client, project_name):
+    """Download files."""
+    out_path = os.path.join(os.path.dirname(__file__), "downloads")
+    log.info(f"Downloading files to {out_path}")
+
+    jms_api = JmsApi(client)
+    project = jms_api.get_project_by_name(name=project_name)
+    project = jms_api.get_project(id=project.id)
+
+    log.info(f"Project id: {project.id}")
+    project_api = ProjectApi(client, project.id)
+
+    jobs = project_api.get_jobs(eval_status="evaluated", fields=["id", "values", "elapsed_time"])
+    log.info(f"# evaluated jobs: {len(jobs)}")
+    num = len(jobs)
+
+    log.info(
+        f"=== Example 1: Downloading output files of {num} jobs using ProjectApi.download_file()"
+    )
+    for job in jobs[0:num]:
+        log.info(f"Job {job.id}")
+        for task in project_api.get_tasks(job_id=job.id):
+            log.info(f"Task {task.id}")
+            files = project_api.get_files(id=task.output_file_ids)
+            for f in files:
+                fpath = os.path.join(out_path, f"task_{task.id}")
+                log.info(f"Download output file {f.evaluation_path} to {fpath}")
+                start = time.process_time()
```

Comment:
Best to use the same timing function you use 2 lines later, otherwise you get meaningless numbers
```suggestion
                start = time.time()
```